plugins/orchestrator-form-api/README.md

```diff
@@ -14,7 +14,7 @@ The decorator will be provided through a factory method that leverages a [Backst
 
 ```typescript
 export type FormDecorator = (
-  FormComponent: React.ComponentType<Partial<FormProps>>,
+  FormComponent: React.ComponentType<FormDecoratorProps>,
```

Comment:
nit: does the user know what is `FormDcoratorProps`? Where to import it from?

Reply:
added it to interface description
also mentioned the full plugin example location (although it isn't ready yet...)